### ISSUE TITLE
Please help: error with translate.worker.js or Xenova/nllb-200-distilled-600M

### ISSUE THREAD

**Issue by ynamite** [jamezmca/free-scribe#2]
Hey there,
Thanks for the lovely ML tutorial. Unfortunately I'm stuck with the following console error after attempting to translate a transcribed result (after choosing a language and clicking translate):
```
ncaught (in promise) SyntaxError: Unexpected token '<', "<!DOCTYPE "... is not valid JSON
    at JSON.parse (<anonymous>)
    at getModelJSON (http://localhost:5173/node_modules/.vite/deps/@xenova_transformers.js?v=d6048e6e:14328:15)
    at async Promise.all (index 0)
    at async loadTokenizer (http://localhost:5173/node_modules/.vite/deps/@xenova_transformers.js?v=d6048e6e:17831:16)
    at async AutoTokenizer.from_pretrained (http://localhost:5173/node_modules/.vite/deps/@xenova_transformers.js?v=d6048e6e:21018:46)
    at async Promise.all (index 0)
    at async loadItems (http://localhost:5173/node_modules/.vite/deps/@xenova_transformers.js?v=d6048e6e:29508:3)
    at async pipeline (http://localhost:5173/node_modules/.vite/deps/@xenova_transformers.js?v=d6048e6e:29469:19)
    at async http://localhost:5173/src/utils/translate.worker.js?worker_file&type=module:22:22
```

If I log out the last progress update before the error, I get the following, which looks good to me:
```
{
    "status": "progress",
    "name": "Xenova/nllb-200-distilled-600M",
    "file": "tokenizer.json",
    "progress": 100,
    "loaded": 1762,
    "total": 1762
}
```

I do get a `done` but never a `complete` status from the worker.

**Comment by ynamite** [comment]
This happens regardless of whether I'm in a private or a normal browser window

**Comment by ynamite** [comment]
I've also tried downgrading xenova from 2.17.2 to 2.4.2, the same version your repo uses.
While there's no error message with this older version, the worker just never stops running at all and never reaches the `complete` status 🤷

**Comment by ynamite** [comment]
I've cloned your repo and tried it in a private tab, same problem. Translation never completes, it just runs and runs.

**Comment by Hemanth12-git** [comment]
I am too getting the same error on the console. Did you come up with a solution for this ? 

I tried many methods but couldnt come up with a solution. If you have already rectified this error, could you kindly post the solution ?

**Comment by ynamite** [comment]
Nope, I eventually gave up and went onto another tutorial

**Comment by jamezmca** [comment]
so adding the following lines as you commented in another issue didn't resolve it? I presume it also failed in all other browser windows (both normal and incognito)?

```
import { pipeline, env } from '@xenova/transformers';
env.allowLocalModels = false;
env.useBrowserCache = false;
```

**Comment by ynamite** [comment]
@jamezmca no, the solution in the other issue gets rid of the `Unexpected token` error, but the translation still never resolves/completes. 

I didn't spend too much time on it to be honest.

**Comment by jamezmca** [comment]
@ynamite ahh that's a pain - it seems to be a super random bug. Cheers for your other solution and I'll have a closer look at this as soon as I get a chance!

**Comment by ynamite** [comment]
@jamezmca ah no worries, thanks for the awesome tutorial, learnt a ton from it.

**Comment by Hemanth12-git** [comment]
Yeah , Kindly share the solution as soon as it is solved because I m doing this as my college's minor project. Haha

**Comment by jasirjaleel** [comment]
Did anyone found out the issue. If so kindly share it

**Comment by Pranava26** [comment]
import { pipeline, env } from '@xenova/transformers';
env.allowLocalModels = false;
env.useBrowserCache = false;

adding this code in translate.worker.js helped me resolve this issue. And yes, its working fine for me in google chrome.
